samples/sqltransport-nhpersistence-outbox-ef/sample.md

```diff
@@ -13,6 +18,7 @@ related:
 ---
 
  1. Make sure you have SQL Server Express installed and accessible as `.\SQLEXPRESS`. Create two databases: `sender` and `receiver`.
+  * You can also use localdb by replacing `SQLEXPRESS` with `(localdb)\v11.0`.
```

Comment:
This is assumed knowledge for anyone working with sql. Also the list of ways u can connect to the various sql types is unbound. And if u wanted to do this properly you would need to apply this to every sample using sql. So i would say for now just remove it.